blocks/init/src/Blocks/custom/featured-posts/components/featured-posts-options.js

```diff
@@ -65,17 +131,20 @@ export const FeaturedPostsOptions = ({ attributes, setAttributes }) => {
 	});
 
 	return (
-		<PanelBody title={__('Featured Post', 'eightshift-frontend-libs')}>
+		<PanelBody title={__('Featured Posts', 'eightshift-frontend-libs')}>
 
-			{postTypeOptions[0] ?
+			{postTypeOptions ?
 				<SelectControl
-					label={__('Post Type', 'eightshift-frontend-libs')}
+					label={__('Filter By Post Type', 'eightshift-frontend-libs')}
+					help={__('Select main section of content to filter from.', 'eightshift-frontend-libs')}
```

Comment:
Can you explain what this means exactly? I'm not sure I understand what this is doing.

Reply:
Select from what post type you want data to be provided

Reply:
Awesome! Put this as a description 😄